From 69bdba4ee94b65041c5fcb8db62f43cac63a5263 Mon Sep 17 00:00:00 2001
From: Konrad Weihmann <kweihmann@outlook.com>
Date: Sun, 26 Jul 2020 16:06:53 +0200
Subject: [PATCH] Bind python to nativepython3

This should ensure that no call is done to maybe non-existing python

Signed-off-by: Konrad Weihmann <kweihmann@outlook.com>
---
 deps/v8/third_party/binutils/detect_v8_host_arch.py             | 2 +-
 deps/v8/third_party/binutils/download.py                        | 2 +-
 .../inspector_protocol/check_protocol_compatibility.py          | 2 +-
 deps/v8/third_party/inspector_protocol/code_generator.py        | 2 +-
 deps/v8/third_party/inspector_protocol/concatenate_protocols.py | 2 +-
 .../third_party/inspector_protocol/convert_protocol_to_json.py  | 2 +-
 deps/v8/third_party/inspector_protocol/roll.py                  | 2 +-
 deps/v8/tools/__init__.py                                       | 2 +-
 deps/v8/tools/adb-d8.py                                         | 2 +-
 deps/v8/tools/android-run.py                                    | 2 +-
 deps/v8/tools/avg.py                                            | 2 +-
 deps/v8/tools/callstats.py                                      | 2 +-
 deps/v8/tools/clusterfuzz/v8_foozzie.py                         | 2 +-
 deps/v8/tools/clusterfuzz/v8_foozzie_test.py                    | 2 +-
 deps/v8/tools/concatenate-files.py                              | 2 +-
 deps/v8/tools/debug_helper/gen-heap-constants.py                | 2 +-
 deps/v8/tools/deprecation_stats.py                              | 2 +-
 deps/v8/tools/dev/gen-tags.py                                   | 2 +-
 deps/v8/tools/dev/gm.py                                         | 2 +-
 deps/v8/tools/dev/v8gen.py                                      | 2 +-
 deps/v8/tools/disasm.py                                         | 2 +-
 deps/v8/tools/dump-cpp.py                                       | 2 +-
 deps/v8/tools/eval_gc_nvp.py                                    | 2 +-
 deps/v8/tools/find-commit-for-patch.py                          | 2 +-
 deps/v8/tools/gc-nvp-to-csv.py                                  | 2 +-
 deps/v8/tools/gc-nvp-trace-processor.py                         | 2 +-
 deps/v8/tools/gcmole/parallel.py                                | 2 +-
 deps/v8/tools/gcmole/run-gcmole.py                              | 2 +-
 deps/v8/tools/gen-inlining-tests.py                             | 2 +-
 deps/v8/tools/gen-keywords-gen-h.py                             | 2 +-
 deps/v8/tools/gen-postmortem-metadata.py                        | 2 +-
 deps/v8/tools/generate-builtins-tests.py                        | 2 +-
 deps/v8/tools/generate-header-include-checks.py                 | 2 +-
 deps/v8/tools/generate_shim_headers/generate_shim_headers.py    | 2 +-
 deps/v8/tools/get_landmines.py                                  | 2 +-
 deps/v8/tools/grokdump.py                                       | 2 +-
 deps/v8/tools/js2c.py                                           | 2 +-
 deps/v8/tools/ll_prof.py                                        | 2 +-
 deps/v8/tools/locs.py                                           | 2 +-
 deps/v8/tools/mb/mb.py                                          | 2 +-
 deps/v8/tools/node/fetch_deps.py                                | 2 +-
 deps/v8/tools/node/node_common.py                               | 2 +-
 deps/v8/tools/objdump-v8                                        | 2 +-
 deps/v8/tools/perf-compare.py                                   | 2 +-
 deps/v8/tools/predictable_wrapper.py                            | 2 +-
 deps/v8/tools/release/auto_push.py                              | 2 +-
 deps/v8/tools/release/auto_roll.py                              | 2 +-
 deps/v8/tools/release/auto_tag.py                               | 2 +-
 deps/v8/tools/release/check_clusterfuzz.py                      | 2 +-
 deps/v8/tools/release/common_includes.py                        | 2 +-
 deps/v8/tools/release/create_release.py                         | 2 +-
 deps/v8/tools/release/filter_build_files.py                     | 2 +-
 deps/v8/tools/release/git_recipes.py                            | 2 +-
 deps/v8/tools/release/merge_to_branch.py                        | 2 +-
 deps/v8/tools/release/mergeinfo.py                              | 2 +-
 deps/v8/tools/release/push_to_candidates.py                     | 2 +-
 deps/v8/tools/release/roll_merge.py                             | 2 +-
 deps/v8/tools/release/script_test.py                            | 2 +-
 deps/v8/tools/release/search_related_commits.py                 | 2 +-
 deps/v8/tools/release/test_mergeinfo.py                         | 2 +-
 deps/v8/tools/release/test_scripts.py                           | 2 +-
 deps/v8/tools/release/test_search_related_commits.py            | 2 +-
 deps/v8/tools/run-clang-tidy.py                                 | 2 +-
 deps/v8/tools/run-num-fuzzer.py                                 | 2 +-
 deps/v8/tools/run-wasm-api-tests.py                             | 2 +-
 deps/v8/tools/run.py                                            | 2 +-
 deps/v8/tools/sanitizers/sancov_formatter.py                    | 2 +-
 deps/v8/tools/sanitizers/sancov_merger.py                       | 2 +-
 deps/v8/tools/sanitizers/sanitize_pcs.py                        | 2 +-
 deps/v8/tools/snapshot/asm_to_inline_asm.py                     | 2 +-
 deps/v8/tools/stats-viewer.py                                   | 2 +-
 deps/v8/tools/testrunner/local/fake_testsuite/testcfg.py        | 2 +-
 deps/v8/tools/testrunner/local/pool.py                          | 2 +-
 deps/v8/tools/testrunner/local/pool_unittest.py                 | 2 +-
 deps/v8/tools/testrunner/local/statusfile_unittest.py           | 2 +-
 deps/v8/tools/testrunner/local/testsuite_unittest.py            | 2 +-
 deps/v8/tools/testrunner/num_fuzzer.py                          | 2 +-
 deps/v8/tools/testrunner/standard_runner.py                     | 2 +-
 deps/v8/tools/testrunner/testproc/shard_unittest.py             | 2 +-
 deps/v8/tools/testrunner/testproc/variant_unittest.py           | 2 +-
 deps/v8/tools/torque/format-torque.py                           | 2 +-
 deps/v8/tools/trace-maps-processor.py                           | 2 +-
 deps/v8/tools/try_perf.py                                       | 2 +-
 deps/v8/tools/unittests/__init__.py                             | 2 +-
 deps/v8/tools/unittests/predictable_wrapper_test.py             | 2 +-
 deps/v8/tools/unittests/run_perf_test.py                        | 2 +-
 deps/v8/tools/unittests/run_tests_test.py                       | 2 +-
 deps/v8/tools/unittests/testdata/d8_mocked1.py                  | 2 +-
 deps/v8/tools/unittests/testdata/d8_mocked2.py                  | 2 +-
 deps/v8/tools/unittests/testdata/predictable_mocked.py          | 2 +-
 deps/v8/tools/unittests/testdata/results_processor.py           | 2 +-
 deps/v8/tools/unittests/v8_presubmit_test.py                    | 2 +-
 deps/v8/tools/update-object-macros-undef.py                     | 2 +-
 deps/v8/tools/v8_presubmit.py                                   | 2 +-
 .../v8/tools/wasm-compilation-hints/inject-compilation-hints.py | 2 +-
 .../wasm-compilation-hints/wasm-objdump-compilation-hints.py    | 2 +-
 deps/v8/tools/wasm-compilation-hints/wasm.py                    | 2 +-
 tools/checkimports.py                                           | 2 +-
 tools/compress_json.py                                          | 2 +-
 tools/configure.d/nodedownload.py                               | 2 +-
 tools/genv8constants.py                                         | 2 +-
 tools/gyp/gyp_main.py                                           | 2 +-
 tools/gyp/pylib/gyp/MSVSSettings_test.py                        | 2 +-
 tools/gyp/pylib/gyp/__init__.py                                 | 2 +-
 tools/gyp/pylib/gyp/common_test.py                              | 2 +-
 tools/gyp/pylib/gyp/easy_xml_test.py                            | 2 +-
 tools/gyp/pylib/gyp/flock_tool.py                               | 2 +-
 tools/gyp/pylib/gyp/generator/msvs_test.py                      | 2 +-
 tools/gyp/pylib/gyp/generator/ninja_test.py                     | 2 +-
 tools/gyp/pylib/gyp/generator/xcode_test.py                     | 2 +-
 tools/gyp/pylib/gyp/input_test.py                               | 2 +-
 tools/gyp/pylib/gyp/mac_tool.py                                 | 2 +-
 tools/gyp/pylib/gyp/win_tool.py                                 | 2 +-
 tools/gyp/setup.py                                              | 2 +-
 tools/gyp/tools/graphviz.py                                     | 2 +-
 tools/gyp/tools/pretty_gyp.py                                   | 2 +-
 tools/gyp/tools/pretty_sln.py                                   | 2 +-
 tools/gyp/tools/pretty_vcproj.py                                | 2 +-
 tools/gyp_node.py                                               | 2 +-
 tools/icu/shrink-icu-src.py                                     | 2 +-
 tools/inspector_protocol/check_protocol_compatibility.py        | 2 +-
 tools/inspector_protocol/code_generator.py                      | 2 +-
 tools/inspector_protocol/concatenate_protocols.py               | 2 +-
 tools/inspector_protocol/convert_protocol_to_json.py            | 2 +-
 tools/inspector_protocol/roll.py                                | 2 +-
 tools/install.py                                                | 2 +-
 tools/js2c.py                                                   | 2 +-
 tools/mkssldef.py                                               | 2 +-
 tools/run-valgrind.py                                           | 2 +-
 tools/specialize_node_d.py                                      | 2 +-
 tools/test.py                                                   | 2 +-
 131 files changed, 131 insertions(+), 131 deletions(-)

diff --git a/deps/v8/third_party/binutils/detect_v8_host_arch.py b/deps/v8/third_party/binutils/detect_v8_host_arch.py
index 3258774..599bb91 100644
--- a/deps/v8/third_party/binutils/detect_v8_host_arch.py
+++ b/deps/v8/third_party/binutils/detect_v8_host_arch.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/third_party/binutils/download.py b/deps/v8/third_party/binutils/download.py
index 7e8df8a..5cffadf 100755
--- a/deps/v8/third_party/binutils/download.py
+++ b/deps/v8/third_party/binutils/download.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/third_party/inspector_protocol/check_protocol_compatibility.py b/deps/v8/third_party/inspector_protocol/check_protocol_compatibility.py
index d2df244..253bb9b 100755
--- a/deps/v8/third_party/inspector_protocol/check_protocol_compatibility.py
+++ b/deps/v8/third_party/inspector_protocol/check_protocol_compatibility.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright (c) 2011 Google Inc. All rights reserved.
 #
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/third_party/inspector_protocol/code_generator.py b/deps/v8/third_party/inspector_protocol/code_generator.py
index 5cf5a30..0f03b2e 100755
--- a/deps/v8/third_party/inspector_protocol/code_generator.py
+++ b/deps/v8/third_party/inspector_protocol/code_generator.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/third_party/inspector_protocol/concatenate_protocols.py b/deps/v8/third_party/inspector_protocol/concatenate_protocols.py
index 42ecc7f..74cf6f3 100755
--- a/deps/v8/third_party/inspector_protocol/concatenate_protocols.py
+++ b/deps/v8/third_party/inspector_protocol/concatenate_protocols.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/third_party/inspector_protocol/convert_protocol_to_json.py b/deps/v8/third_party/inspector_protocol/convert_protocol_to_json.py
index f98bebc..c1480bf 100755
--- a/deps/v8/third_party/inspector_protocol/convert_protocol_to_json.py
+++ b/deps/v8/third_party/inspector_protocol/convert_protocol_to_json.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/third_party/inspector_protocol/roll.py b/deps/v8/third_party/inspector_protocol/roll.py
index ba26519..b2251e5 100755
--- a/deps/v8/third_party/inspector_protocol/roll.py
+++ b/deps/v8/third_party/inspector_protocol/roll.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/__init__.py b/deps/v8/tools/__init__.py
index 3841a86..557256e 100644
--- a/deps/v8/tools/__init__.py
+++ b/deps/v8/tools/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/adb-d8.py b/deps/v8/tools/adb-d8.py
index 4d4390f..ebc1fb6 100755
--- a/deps/v8/tools/adb-d8.py
+++ b/deps/v8/tools/adb-d8.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/android-run.py b/deps/v8/tools/android-run.py
index 66d333a..855847a 100755
--- a/deps/v8/tools/android-run.py
+++ b/deps/v8/tools/android-run.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/avg.py b/deps/v8/tools/avg.py
index 5741acd..a5f7b19 100755
--- a/deps/v8/tools/avg.py
+++ b/deps/v8/tools/avg.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can
diff --git a/deps/v8/tools/callstats.py b/deps/v8/tools/callstats.py
index 7c5bd4a..7fca5be 100755
--- a/deps/v8/tools/callstats.py
+++ b/deps/v8/tools/callstats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/clusterfuzz/v8_foozzie.py b/deps/v8/tools/clusterfuzz/v8_foozzie.py
index ff481e9..4624cb7 100755
--- a/deps/v8/tools/clusterfuzz/v8_foozzie.py
+++ b/deps/v8/tools/clusterfuzz/v8_foozzie.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/clusterfuzz/v8_foozzie_test.py b/deps/v8/tools/clusterfuzz/v8_foozzie_test.py
index 43b65e8..13dc994 100755
--- a/deps/v8/tools/clusterfuzz/v8_foozzie_test.py
+++ b/deps/v8/tools/clusterfuzz/v8_foozzie_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/concatenate-files.py b/deps/v8/tools/concatenate-files.py
index a5dbe45..41f38d8 100644
--- a/deps/v8/tools/concatenate-files.py
+++ b/deps/v8/tools/concatenate-files.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/debug_helper/gen-heap-constants.py b/deps/v8/tools/debug_helper/gen-heap-constants.py
index 0fd575a..a064f4e 100644
--- a/deps/v8/tools/debug_helper/gen-heap-constants.py
+++ b/deps/v8/tools/debug_helper/gen-heap-constants.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/deprecation_stats.py b/deps/v8/tools/deprecation_stats.py
index 628eebc..83db036 100755
--- a/deps/v8/tools/deprecation_stats.py
+++ b/deps/v8/tools/deprecation_stats.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/dev/gen-tags.py b/deps/v8/tools/dev/gen-tags.py
index a478ee3..cf9adab 100755
--- a/deps/v8/tools/dev/gen-tags.py
+++ b/deps/v8/tools/dev/gen-tags.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/dev/gm.py b/deps/v8/tools/dev/gm.py
index 0e01f4f..3685eac 100755
--- a/deps/v8/tools/dev/gm.py
+++ b/deps/v8/tools/dev/gm.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/dev/v8gen.py b/deps/v8/tools/dev/v8gen.py
index 0b6e1d1..98c1b8e 100755
--- a/deps/v8/tools/dev/v8gen.py
+++ b/deps/v8/tools/dev/v8gen.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/disasm.py b/deps/v8/tools/disasm.py
index a91d0db..951a600 100644
--- a/deps/v8/tools/disasm.py
+++ b/deps/v8/tools/disasm.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2011 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/dump-cpp.py b/deps/v8/tools/dump-cpp.py
index 31dc053..5ca200b 100644
--- a/deps/v8/tools/dump-cpp.py
+++ b/deps/v8/tools/dump-cpp.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/eval_gc_nvp.py b/deps/v8/tools/eval_gc_nvp.py
index 222ebef..b70f13a 100755
--- a/deps/v8/tools/eval_gc_nvp.py
+++ b/deps/v8/tools/eval_gc_nvp.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/find-commit-for-patch.py b/deps/v8/tools/find-commit-for-patch.py
index cca1f40..4efbf9e 100755
--- a/deps/v8/tools/find-commit-for-patch.py
+++ b/deps/v8/tools/find-commit-for-patch.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/gc-nvp-to-csv.py b/deps/v8/tools/gc-nvp-to-csv.py
index b3ad374..5f411ae 100755
--- a/deps/v8/tools/gc-nvp-to-csv.py
+++ b/deps/v8/tools/gc-nvp-to-csv.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/gc-nvp-trace-processor.py b/deps/v8/tools/gc-nvp-trace-processor.py
index 75d50b1..1374a77 100755
--- a/deps/v8/tools/gc-nvp-trace-processor.py
+++ b/deps/v8/tools/gc-nvp-trace-processor.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2010 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/gcmole/parallel.py b/deps/v8/tools/gcmole/parallel.py
index 7ff95cc..3e6d0f7 100755
--- a/deps/v8/tools/gcmole/parallel.py
+++ b/deps/v8/tools/gcmole/parallel.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/gcmole/run-gcmole.py b/deps/v8/tools/gcmole/run-gcmole.py
index 6f2a091..2783b32 100755
--- a/deps/v8/tools/gcmole/run-gcmole.py
+++ b/deps/v8/tools/gcmole/run-gcmole.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/gen-inlining-tests.py b/deps/v8/tools/gen-inlining-tests.py
index 400386c..f8cae03 100644
--- a/deps/v8/tools/gen-inlining-tests.py
+++ b/deps/v8/tools/gen-inlining-tests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/gen-keywords-gen-h.py b/deps/v8/tools/gen-keywords-gen-h.py
index 02750dc..7648cdc 100755
--- a/deps/v8/tools/gen-keywords-gen-h.py
+++ b/deps/v8/tools/gen-keywords-gen-h.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/gen-postmortem-metadata.py b/deps/v8/tools/gen-postmortem-metadata.py
index 021a2f2..1231151 100644
--- a/deps/v8/tools/gen-postmortem-metadata.py
+++ b/deps/v8/tools/gen-postmortem-metadata.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
diff --git a/deps/v8/tools/generate-builtins-tests.py b/deps/v8/tools/generate-builtins-tests.py
index 4380f91..a46965b 100755
--- a/deps/v8/tools/generate-builtins-tests.py
+++ b/deps/v8/tools/generate-builtins-tests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/generate-header-include-checks.py b/deps/v8/tools/generate-header-include-checks.py
index fa18d85..8067408 100755
--- a/deps/v8/tools/generate-header-include-checks.py
+++ b/deps/v8/tools/generate-header-include-checks.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # vim:fenc=utf-8:shiftwidth=2
 
 # Copyright 2018 the V8 project authors. All rights reserved.
diff --git a/deps/v8/tools/generate_shim_headers/generate_shim_headers.py b/deps/v8/tools/generate_shim_headers/generate_shim_headers.py
index d0e6d06..a71720e 100755
--- a/deps/v8/tools/generate_shim_headers/generate_shim_headers.py
+++ b/deps/v8/tools/generate_shim_headers/generate_shim_headers.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2013 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/get_landmines.py b/deps/v8/tools/get_landmines.py
index bf8efa5..4d62bf5 100755
--- a/deps/v8/tools/get_landmines.py
+++ b/deps/v8/tools/get_landmines.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/grokdump.py b/deps/v8/tools/grokdump.py
index 773622d..0cce695 100755
--- a/deps/v8/tools/grokdump.py
+++ b/deps/v8/tools/grokdump.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/js2c.py b/deps/v8/tools/js2c.py
index b94f3ad..3ef4d93 100755
--- a/deps/v8/tools/js2c.py
+++ b/deps/v8/tools/js2c.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/ll_prof.py b/deps/v8/tools/ll_prof.py
index 4499874..76564ef 100755
--- a/deps/v8/tools/ll_prof.py
+++ b/deps/v8/tools/ll_prof.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/locs.py b/deps/v8/tools/locs.py
index bd1609d..3ebb995 100755
--- a/deps/v8/tools/locs.py
+++ b/deps/v8/tools/locs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/mb/mb.py b/deps/v8/tools/mb/mb.py
index f66f82a..eea6e3c 100755
--- a/deps/v8/tools/mb/mb.py
+++ b/deps/v8/tools/mb/mb.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Copyright 2015 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/node/fetch_deps.py b/deps/v8/tools/node/fetch_deps.py
index ee5b629..a859c73 100755
--- a/deps/v8/tools/node/fetch_deps.py
+++ b/deps/v8/tools/node/fetch_deps.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/node/node_common.py b/deps/v8/tools/node/node_common.py
index 2efb218..7aa4558 100755
--- a/deps/v8/tools/node/node_common.py
+++ b/deps/v8/tools/node/node_common.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/objdump-v8 b/deps/v8/tools/objdump-v8
index dc7310a..001b057 100755
--- a/deps/v8/tools/objdump-v8
+++ b/deps/v8/tools/objdump-v8
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/perf-compare.py b/deps/v8/tools/perf-compare.py
index 744f6aa..24f4f43 100755
--- a/deps/v8/tools/perf-compare.py
+++ b/deps/v8/tools/perf-compare.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/predictable_wrapper.py b/deps/v8/tools/predictable_wrapper.py
index 31b62c8..372e8d0 100644
--- a/deps/v8/tools/predictable_wrapper.py
+++ b/deps/v8/tools/predictable_wrapper.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/auto_push.py b/deps/v8/tools/release/auto_push.py
index 4cb9687..fe88d27 100755
--- a/deps/v8/tools/release/auto_push.py
+++ b/deps/v8/tools/release/auto_push.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2013 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/auto_roll.py b/deps/v8/tools/release/auto_roll.py
index 829aefb..20ec27d 100755
--- a/deps/v8/tools/release/auto_roll.py
+++ b/deps/v8/tools/release/auto_roll.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/auto_tag.py b/deps/v8/tools/release/auto_tag.py
index fddefed..188d7e7 100755
--- a/deps/v8/tools/release/auto_tag.py
+++ b/deps/v8/tools/release/auto_tag.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/check_clusterfuzz.py b/deps/v8/tools/release/check_clusterfuzz.py
index 021cd55..88d88e4 100755
--- a/deps/v8/tools/release/check_clusterfuzz.py
+++ b/deps/v8/tools/release/check_clusterfuzz.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/common_includes.py b/deps/v8/tools/release/common_includes.py
index af4709b..074935f 100644
--- a/deps/v8/tools/release/common_includes.py
+++ b/deps/v8/tools/release/common_includes.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2013 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/create_release.py b/deps/v8/tools/release/create_release.py
index cbf07f4..16960ef 100755
--- a/deps/v8/tools/release/create_release.py
+++ b/deps/v8/tools/release/create_release.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/filter_build_files.py b/deps/v8/tools/release/filter_build_files.py
index 1afabe8..cda38e9 100755
--- a/deps/v8/tools/release/filter_build_files.py
+++ b/deps/v8/tools/release/filter_build_files.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/git_recipes.py b/deps/v8/tools/release/git_recipes.py
index 0997e0b..195057e 100644
--- a/deps/v8/tools/release/git_recipes.py
+++ b/deps/v8/tools/release/git_recipes.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/merge_to_branch.py b/deps/v8/tools/release/merge_to_branch.py
index c959429..b95b3fb 100755
--- a/deps/v8/tools/release/merge_to_branch.py
+++ b/deps/v8/tools/release/merge_to_branch.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/mergeinfo.py b/deps/v8/tools/release/mergeinfo.py
index bed7441..dc0c89d 100755
--- a/deps/v8/tools/release/mergeinfo.py
+++ b/deps/v8/tools/release/mergeinfo.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/push_to_candidates.py b/deps/v8/tools/release/push_to_candidates.py
index c706896..558ce8e 100755
--- a/deps/v8/tools/release/push_to_candidates.py
+++ b/deps/v8/tools/release/push_to_candidates.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2013 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/roll_merge.py b/deps/v8/tools/release/roll_merge.py
index 2f03abb..ded336f 100755
--- a/deps/v8/tools/release/roll_merge.py
+++ b/deps/v8/tools/release/roll_merge.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/script_test.py b/deps/v8/tools/release/script_test.py
index 0f345b7..6cddbc2 100755
--- a/deps/v8/tools/release/script_test.py
+++ b/deps/v8/tools/release/script_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/search_related_commits.py b/deps/v8/tools/release/search_related_commits.py
index e6e52d2..66bc4a5 100755
--- a/deps/v8/tools/release/search_related_commits.py
+++ b/deps/v8/tools/release/search_related_commits.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/test_mergeinfo.py b/deps/v8/tools/release/test_mergeinfo.py
index f8619bb..c5c3ef4 100755
--- a/deps/v8/tools/release/test_mergeinfo.py
+++ b/deps/v8/tools/release/test_mergeinfo.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/release/test_scripts.py b/deps/v8/tools/release/test_scripts.py
index 62158d9..183f758 100755
--- a/deps/v8/tools/release/test_scripts.py
+++ b/deps/v8/tools/release/test_scripts.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2013 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions are
diff --git a/deps/v8/tools/release/test_search_related_commits.py b/deps/v8/tools/release/test_search_related_commits.py
index cf61236..f443f49 100755
--- a/deps/v8/tools/release/test_search_related_commits.py
+++ b/deps/v8/tools/release/test_search_related_commits.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2015 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/run-clang-tidy.py b/deps/v8/tools/run-clang-tidy.py
index aee1b40..a4683bb 100755
--- a/deps/v8/tools/run-clang-tidy.py
+++ b/deps/v8/tools/run-clang-tidy.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/run-num-fuzzer.py b/deps/v8/tools/run-num-fuzzer.py
index 9b5a065..4410190 100755
--- a/deps/v8/tools/run-num-fuzzer.py
+++ b/deps/v8/tools/run-num-fuzzer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/run-wasm-api-tests.py b/deps/v8/tools/run-wasm-api-tests.py
index ff37c8a..bd4c71f 100755
--- a/deps/v8/tools/run-wasm-api-tests.py
+++ b/deps/v8/tools/run-wasm-api-tests.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/run.py b/deps/v8/tools/run.py
index 5a656e1..fc748a0 100755
--- a/deps/v8/tools/run.py
+++ b/deps/v8/tools/run.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/sanitizers/sancov_formatter.py b/deps/v8/tools/sanitizers/sancov_formatter.py
index b66bfed..d159e9a 100755
--- a/deps/v8/tools/sanitizers/sancov_formatter.py
+++ b/deps/v8/tools/sanitizers/sancov_formatter.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/sanitizers/sancov_merger.py b/deps/v8/tools/sanitizers/sancov_merger.py
index 6fd2eb2..d553a4b 100755
--- a/deps/v8/tools/sanitizers/sancov_merger.py
+++ b/deps/v8/tools/sanitizers/sancov_merger.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/sanitizers/sanitize_pcs.py b/deps/v8/tools/sanitizers/sanitize_pcs.py
index a1e3a1d..af645fc 100755
--- a/deps/v8/tools/sanitizers/sanitize_pcs.py
+++ b/deps/v8/tools/sanitizers/sanitize_pcs.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/snapshot/asm_to_inline_asm.py b/deps/v8/tools/snapshot/asm_to_inline_asm.py
index ad8fdcb..c64a02c 100644
--- a/deps/v8/tools/snapshot/asm_to_inline_asm.py
+++ b/deps/v8/tools/snapshot/asm_to_inline_asm.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/stats-viewer.py b/deps/v8/tools/stats-viewer.py
index dd9d2c2..e9a9c76 100755
--- a/deps/v8/tools/stats-viewer.py
+++ b/deps/v8/tools/stats-viewer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2008 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/testrunner/local/fake_testsuite/testcfg.py b/deps/v8/tools/testrunner/local/fake_testsuite/testcfg.py
index 28de737..a89a44e 100644
--- a/deps/v8/tools/testrunner/local/fake_testsuite/testcfg.py
+++ b/deps/v8/tools/testrunner/local/fake_testsuite/testcfg.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/local/pool.py b/deps/v8/tools/testrunner/local/pool.py
index 9defdd3..cf12f6a 100644
--- a/deps/v8/tools/testrunner/local/pool.py
+++ b/deps/v8/tools/testrunner/local/pool.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/local/pool_unittest.py b/deps/v8/tools/testrunner/local/pool_unittest.py
index 240cd56..14855bc 100755
--- a/deps/v8/tools/testrunner/local/pool_unittest.py
+++ b/deps/v8/tools/testrunner/local/pool_unittest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/local/statusfile_unittest.py b/deps/v8/tools/testrunner/local/statusfile_unittest.py
index e8d5ff9..d8bea37 100755
--- a/deps/v8/tools/testrunner/local/statusfile_unittest.py
+++ b/deps/v8/tools/testrunner/local/statusfile_unittest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/local/testsuite_unittest.py b/deps/v8/tools/testrunner/local/testsuite_unittest.py
index b74fef1..d74c98a 100755
--- a/deps/v8/tools/testrunner/local/testsuite_unittest.py
+++ b/deps/v8/tools/testrunner/local/testsuite_unittest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/num_fuzzer.py b/deps/v8/tools/testrunner/num_fuzzer.py
index d4e92a6..0ec581a 100755
--- a/deps/v8/tools/testrunner/num_fuzzer.py
+++ b/deps/v8/tools/testrunner/num_fuzzer.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/testrunner/standard_runner.py b/deps/v8/tools/testrunner/standard_runner.py
index 51e7860..db30bc5 100755
--- a/deps/v8/tools/testrunner/standard_runner.py
+++ b/deps/v8/tools/testrunner/standard_runner.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/testrunner/testproc/shard_unittest.py b/deps/v8/tools/testrunner/testproc/shard_unittest.py
index 33a094e..aead259 100755
--- a/deps/v8/tools/testrunner/testproc/shard_unittest.py
+++ b/deps/v8/tools/testrunner/testproc/shard_unittest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/testrunner/testproc/variant_unittest.py b/deps/v8/tools/testrunner/testproc/variant_unittest.py
index 56e28c8..12406db 100755
--- a/deps/v8/tools/testrunner/testproc/variant_unittest.py
+++ b/deps/v8/tools/testrunner/testproc/variant_unittest.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/torque/format-torque.py b/deps/v8/tools/torque/format-torque.py
index 2e04e65..2b91055 100755
--- a/deps/v8/tools/torque/format-torque.py
+++ b/deps/v8/tools/torque/format-torque.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # -*- coding: utf-8 -*-
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/trace-maps-processor.py b/deps/v8/tools/trace-maps-processor.py
index 4a29eab..9c9e368 100755
--- a/deps/v8/tools/trace-maps-processor.py
+++ b/deps/v8/tools/trace-maps-processor.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/try_perf.py b/deps/v8/tools/try_perf.py
index 2c9c382..ee9d901 100755
--- a/deps/v8/tools/try_perf.py
+++ b/deps/v8/tools/try_perf.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/__init__.py b/deps/v8/tools/unittests/__init__.py
index 3841a86..557256e 100644
--- a/deps/v8/tools/unittests/__init__.py
+++ b/deps/v8/tools/unittests/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/predictable_wrapper_test.py b/deps/v8/tools/unittests/predictable_wrapper_test.py
index c085fb8..e8d05b4 100755
--- a/deps/v8/tools/unittests/predictable_wrapper_test.py
+++ b/deps/v8/tools/unittests/predictable_wrapper_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/run_perf_test.py b/deps/v8/tools/unittests/run_perf_test.py
index 083d224..c175d54 100755
--- a/deps/v8/tools/unittests/run_perf_test.py
+++ b/deps/v8/tools/unittests/run_perf_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2014 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/run_tests_test.py b/deps/v8/tools/unittests/run_tests_test.py
index 93b10f5..6ac9a80 100755
--- a/deps/v8/tools/unittests/run_tests_test.py
+++ b/deps/v8/tools/unittests/run_tests_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/testdata/d8_mocked1.py b/deps/v8/tools/unittests/testdata/d8_mocked1.py
index ff330af..4f29a8f 100644
--- a/deps/v8/tools/unittests/testdata/d8_mocked1.py
+++ b/deps/v8/tools/unittests/testdata/d8_mocked1.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/testdata/d8_mocked2.py b/deps/v8/tools/unittests/testdata/d8_mocked2.py
index 3630462..1fa979f 100644
--- a/deps/v8/tools/unittests/testdata/d8_mocked2.py
+++ b/deps/v8/tools/unittests/testdata/d8_mocked2.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/testdata/predictable_mocked.py b/deps/v8/tools/unittests/testdata/predictable_mocked.py
index b9e73f6..b1ab28b 100644
--- a/deps/v8/tools/unittests/testdata/predictable_mocked.py
+++ b/deps/v8/tools/unittests/testdata/predictable_mocked.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/testdata/results_processor.py b/deps/v8/tools/unittests/testdata/results_processor.py
index d8c5ad9..da2d3ee 100644
--- a/deps/v8/tools/unittests/testdata/results_processor.py
+++ b/deps/v8/tools/unittests/testdata/results_processor.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/unittests/v8_presubmit_test.py b/deps/v8/tools/unittests/v8_presubmit_test.py
index 2c66d18..9cd09de 100755
--- a/deps/v8/tools/unittests/v8_presubmit_test.py
+++ b/deps/v8/tools/unittests/v8_presubmit_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/deps/v8/tools/update-object-macros-undef.py b/deps/v8/tools/update-object-macros-undef.py
index 866fdb6..f6efa8c 100755
--- a/deps/v8/tools/update-object-macros-undef.py
+++ b/deps/v8/tools/update-object-macros-undef.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env nativepython33
 
 # Copyright 2018 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/deps/v8/tools/v8_presubmit.py b/deps/v8/tools/v8_presubmit.py
index 346fc9a..cc227c0 100755
--- a/deps/v8/tools/v8_presubmit.py
+++ b/deps/v8/tools/v8_presubmit.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2012 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/deps/v8/tools/wasm-compilation-hints/inject-compilation-hints.py b/deps/v8/tools/wasm-compilation-hints/inject-compilation-hints.py
index fd4b65b..4fd9e93 100755
--- a/deps/v8/tools/wasm-compilation-hints/inject-compilation-hints.py
+++ b/deps/v8/tools/wasm-compilation-hints/inject-compilation-hints.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be found
diff --git a/deps/v8/tools/wasm-compilation-hints/wasm-objdump-compilation-hints.py b/deps/v8/tools/wasm-compilation-hints/wasm-objdump-compilation-hints.py
index a762bd7..8a33252 100755
--- a/deps/v8/tools/wasm-compilation-hints/wasm-objdump-compilation-hints.py
+++ b/deps/v8/tools/wasm-compilation-hints/wasm-objdump-compilation-hints.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be found
diff --git a/deps/v8/tools/wasm-compilation-hints/wasm.py b/deps/v8/tools/wasm-compilation-hints/wasm.py
index ae3d084..d8d62a0 100644
--- a/deps/v8/tools/wasm-compilation-hints/wasm.py
+++ b/deps/v8/tools/wasm-compilation-hints/wasm.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2019 the V8 project authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be found
diff --git a/tools/checkimports.py b/tools/checkimports.py
index 609a75f..a86ec0c 100755
--- a/tools/checkimports.py
+++ b/tools/checkimports.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 from __future__ import print_function
 import glob
diff --git a/tools/compress_json.py b/tools/compress_json.py
index dfe6406..1bde491 100644
--- a/tools/compress_json.py
+++ b/tools/compress_json.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 import json
 import struct
diff --git a/tools/configure.d/nodedownload.py b/tools/configure.d/nodedownload.py
index 515704e..7ebfb87 100644
--- a/tools/configure.d/nodedownload.py
+++ b/tools/configure.d/nodedownload.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Moved some utilities here from ../../configure
 
 from __future__ import print_function
diff --git a/tools/genv8constants.py b/tools/genv8constants.py
index 8ab2ed8..ea02aeb 100755
--- a/tools/genv8constants.py
+++ b/tools/genv8constants.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 #
 # genv8constants.py output_file libv8_base.a
diff --git a/tools/gyp/gyp_main.py b/tools/gyp/gyp_main.py
index f738e80..2711427 100755
--- a/tools/gyp/gyp_main.py
+++ b/tools/gyp/gyp_main.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2009 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/MSVSSettings_test.py b/tools/gyp/pylib/gyp/MSVSSettings_test.py
index 77b79e6..d3d0469 100755
--- a/tools/gyp/pylib/gyp/MSVSSettings_test.py
+++ b/tools/gyp/pylib/gyp/MSVSSettings_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/__init__.py b/tools/gyp/pylib/gyp/__init__.py
index dee8340..2fb5a5a 100755
--- a/tools/gyp/pylib/gyp/__init__.py
+++ b/tools/gyp/pylib/gyp/__init__.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/common_test.py b/tools/gyp/pylib/gyp/common_test.py
index b75bbb8..039402b 100755
--- a/tools/gyp/pylib/gyp/common_test.py
+++ b/tools/gyp/pylib/gyp/common_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/easy_xml_test.py b/tools/gyp/pylib/gyp/easy_xml_test.py
index 664b538..15cd1d5 100755
--- a/tools/gyp/pylib/gyp/easy_xml_test.py
+++ b/tools/gyp/pylib/gyp/easy_xml_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2011 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/flock_tool.py b/tools/gyp/pylib/gyp/flock_tool.py
index 81fb79d..0214919 100755
--- a/tools/gyp/pylib/gyp/flock_tool.py
+++ b/tools/gyp/pylib/gyp/flock_tool.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright (c) 2011 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/gyp/pylib/gyp/generator/msvs_test.py b/tools/gyp/pylib/gyp/generator/msvs_test.py
index 1b0cdd1..39257ec 100755
--- a/tools/gyp/pylib/gyp/generator/msvs_test.py
+++ b/tools/gyp/pylib/gyp/generator/msvs_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/gyp/pylib/gyp/generator/ninja_test.py b/tools/gyp/pylib/gyp/generator/ninja_test.py
index c8adc25..11dbf2d 100644
--- a/tools/gyp/pylib/gyp/generator/ninja_test.py
+++ b/tools/gyp/pylib/gyp/generator/ninja_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/generator/xcode_test.py b/tools/gyp/pylib/gyp/generator/xcode_test.py
index 260324a..4e01c28 100644
--- a/tools/gyp/pylib/gyp/generator/xcode_test.py
+++ b/tools/gyp/pylib/gyp/generator/xcode_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2013 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/input_test.py b/tools/gyp/pylib/gyp/input_test.py
index 1bc5e3d..88293d1 100755
--- a/tools/gyp/pylib/gyp/input_test.py
+++ b/tools/gyp/pylib/gyp/input_test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright 2013 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/pylib/gyp/mac_tool.py b/tools/gyp/pylib/gyp/mac_tool.py
index 781a863..4a89818 100755
--- a/tools/gyp/pylib/gyp/mac_tool.py
+++ b/tools/gyp/pylib/gyp/mac_tool.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/gyp/pylib/gyp/win_tool.py b/tools/gyp/pylib/gyp/win_tool.py
index cfdacb0..858b02c 100755
--- a/tools/gyp/pylib/gyp/win_tool.py
+++ b/tools/gyp/pylib/gyp/win_tool.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/setup.py b/tools/gyp/setup.py
index 75a4255..b4e27d2 100755
--- a/tools/gyp/setup.py
+++ b/tools/gyp/setup.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2009 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/tools/graphviz.py b/tools/gyp/tools/graphviz.py
index 538b059..c784e7a 100755
--- a/tools/gyp/tools/graphviz.py
+++ b/tools/gyp/tools/graphviz.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2011 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/tools/pretty_gyp.py b/tools/gyp/tools/pretty_gyp.py
index 633048a..f5b8802 100755
--- a/tools/gyp/tools/pretty_gyp.py
+++ b/tools/gyp/tools/pretty_gyp.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/tools/pretty_sln.py b/tools/gyp/tools/pretty_sln.py
index 196566f..36de5ab 100755
--- a/tools/gyp/tools/pretty_sln.py
+++ b/tools/gyp/tools/pretty_sln.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp/tools/pretty_vcproj.py b/tools/gyp/tools/pretty_vcproj.py
index 3212626..73baa65 100755
--- a/tools/gyp/tools/pretty_vcproj.py
+++ b/tools/gyp/tools/pretty_vcproj.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 # Copyright (c) 2012 Google Inc. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
diff --git a/tools/gyp_node.py b/tools/gyp_node.py
index 831d2ef..6f0ee54 100755
--- a/tools/gyp_node.py
+++ b/tools/gyp_node.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 from __future__ import print_function
 import os
 import sys
diff --git a/tools/icu/shrink-icu-src.py b/tools/icu/shrink-icu-src.py
index 0df16cd..03e07b2 100644
--- a/tools/icu/shrink-icu-src.py
+++ b/tools/icu/shrink-icu-src.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 from __future__ import print_function
 import optparse
 import os
diff --git a/tools/inspector_protocol/check_protocol_compatibility.py b/tools/inspector_protocol/check_protocol_compatibility.py
index d2df244..253bb9b 100755
--- a/tools/inspector_protocol/check_protocol_compatibility.py
+++ b/tools/inspector_protocol/check_protocol_compatibility.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright (c) 2011 Google Inc. All rights reserved.
 #
 # Redistribution and use in source and binary forms, with or without
diff --git a/tools/inspector_protocol/code_generator.py b/tools/inspector_protocol/code_generator.py
index c1f78dc..76f1241 100755
--- a/tools/inspector_protocol/code_generator.py
+++ b/tools/inspector_protocol/code_generator.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/inspector_protocol/concatenate_protocols.py b/tools/inspector_protocol/concatenate_protocols.py
index e9f448e..8b9702e 100755
--- a/tools/inspector_protocol/concatenate_protocols.py
+++ b/tools/inspector_protocol/concatenate_protocols.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2016 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/inspector_protocol/convert_protocol_to_json.py b/tools/inspector_protocol/convert_protocol_to_json.py
index 835e638..0a67d7a 100755
--- a/tools/inspector_protocol/convert_protocol_to_json.py
+++ b/tools/inspector_protocol/convert_protocol_to_json.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2017 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/inspector_protocol/roll.py b/tools/inspector_protocol/roll.py
index abe636e..15b9f81 100644
--- a/tools/inspector_protocol/roll.py
+++ b/tools/inspector_protocol/roll.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 # Copyright 2019 The Chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
diff --git a/tools/install.py b/tools/install.py
index fe4723b..131ddd7 100755
--- a/tools/install.py
+++ b/tools/install.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 from __future__ import print_function
 
diff --git a/tools/js2c.py b/tools/js2c.py
index 195e6a6..8ffaaec 100755
--- a/tools/js2c.py
+++ b/tools/js2c.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2006-2008 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/tools/mkssldef.py b/tools/mkssldef.py
index 4768d1f..ed57764 100755
--- a/tools/mkssldef.py
+++ b/tools/mkssldef.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 from __future__ import print_function
 import re
diff --git a/tools/run-valgrind.py b/tools/run-valgrind.py
index bfb67d3..388f01e 100755
--- a/tools/run-valgrind.py
+++ b/tools/run-valgrind.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2009 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
diff --git a/tools/specialize_node_d.py b/tools/specialize_node_d.py
index 825047c..ee0decd 100755
--- a/tools/specialize_node_d.py
+++ b/tools/specialize_node_d.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 
 #
 # specialize_node_d.py output_file src/node.d flavor arch
diff --git a/tools/test.py b/tools/test.py
index 811db91..20adfd4 100755
--- a/tools/test.py
+++ b/tools/test.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env nativepython3
 #
 # Copyright 2008 the V8 project authors. All rights reserved.
 # Redistribution and use in source and binary forms, with or without
-- 
2.20.1

